hack/run-echo-checkup

```diff
@@ -29,8 +27,6 @@ while true; do
     job_failed=true
     break
   fi
-
-  sleep $WAIT_PERIOD_SEC
```

Comment:
If you remove this sleep, the loop will run for no reason as fast as it can.

Reply:
Oh I got it wrong, I will restore it sorry.